src/shared/markdown-parser.ts

```diff
@@ -256,7 +256,12 @@ export function createDefaultMarkdownItInstance(
         linkify: true, // automatically link plain URLs
     });
 
-    if (!features.tables) {
+    if (features.tables) {
+        defaultMarkdownItInstance.enable("table");
+        defaultMarkdownItInstance.renderer.rules.table_open = () => '<div class="s-table-container"><table class="s-table">';
+        defaultMarkdownItInstance.renderer.rules.table_close = () => '</table></div>';
+    }
+    else {
```

Comment:
I'd really rather not start sticking random render rules into this file. My vote is to either pull in the Stack Overflow renderer wholesale or not at all. As it stands, the renderer used currently is unmodified, so it doesn't support things like taglinks or spoilers either.

Reply:
What would "pulling in the renderer wholesale" entail? We are going to be looking to have parity between this preview and the existing one anyway.

Reply:
Yes, that's how the preview is designed currently. See the commonmark preview settings [here](https://github.com/StackExchange/Stacks-Editor/blob/main/src/commonmark/editor.ts#L46-L52) for more details.